index.html

```diff
@@ -1,19 +1,140 @@
-</head>
-<body>
-
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
-</body>
+    <!-- Header -->
+    <header class="header">
+      <img src="img/karma-logo.svg" alt="karma-logo" class="karma-icon" />
+      <img src="img/menu-hamburger.svg" class="menu-hamburger" />
+      <nav>
+        <ul class="nav">
+          <li class="nav-list">
+            <a href="#meet-karma" class="nav-link meet-karma">Meet Karma</a>
+          </li>
+          <li class="nav-list">
+            <a href="#how-it-works" class="nav-link">How it works</a>
+          </li>
+          <li class="nav-list">
+            <a href="store.html" class="nav-link">Store</a>
+          </li>
+          <li class="nav-list">
+            <a href="#blog" class="nav-link">Blog</a>
+          </li>
+          <li class="nav-list">
+            <a href="#help" class="nav-link">Help</a>
+          </li>
+          <li class="nav-list">
+            <a href="#login" class="nav-link">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- Main Content -->
+    <main class="main">
+      <!-- Intro subsection -->
+      <section class="intro">
+        <h1 class="intro-title">Introducing Karma</h1>
+        <p class="description">Bring WiFi with you, everywhere you go.</p>
+        <button class="learn-more-button">Learn More</button>
+      </section>
+
+      <!-- Info subsection -->
+      <section class="info">
+        <p class="info-text">Everyone needs a little Karma.</p>
+        <div class="services">
+          <div class="service">
+            <img
+              src="img/icon-devices.svg"
+              alt="devices"
+              class="service-icon"
+            />
+            <p class="service-text">Internet for all devices</p>
+          </div>
+
+          <div class="service">
+            <img src="img/icon-coffee.svg" alt="coffee" class="service-icon" />
+            <p class="service-text">Boost your productivity</p>
+          </div>
+
+          <div class="service">
+            <img src="img/icon-refill.svg" alt="fuel" class="service-icon" />
+            <p class="service-text">Pay as You Go</p>
+          </div>
+        </div>
+      </section>
+
+      <!-- Quote subsection -->
+      <section class="quote">
+        <!-- Used flexbox so that image and quote can go side by side -->
+        <div class="quote-image-box"></div>
+
+        <div class="quote-text-box">
+          <p class="quote-text">
+            <i>
```

Comment:
You could use the `blockquote` tag here instead, and just make it italic in your CSS,